tools/test_prover.py

```
@@ -388,6 +389,84 @@ def test_a_search_with_witnesses_also_reports_termination(self) -> None:
         self.assertRegex(output, TERMINATION_LINE)
 
 
+class SurveyTests(ProverTestCase):
+    """Counting the blind spots, not stopping at the first."""
+
+    def survey(self, grammar: str, level: int, examples: int = 3) -> tuple[int, str]:
+        path = self.directory / "grammar.mly"
+        path.write_text(grammar, encoding="utf-8")
+        result = subprocess.run(
```

Ruff (0.16.1)

[error] `subprocess` call: check for execution of untrusted input

(S603)